### PR TITLE
Adding ignored paths to GH Action to match prettierignore for prettier changes

### LINKED ISSUE #1994: Not running prettier GH Action in unrelated changes
There are paths ignored by prettier as by https://github.com/w3c/wot-thing-description/blob/main/.prettierignore . However, a GH Action will still run and waste time. We should simply align the action to ignore the same way.

### PR DESCRIPTION
**Pull request by egekorkan**
fixes #1994 

Mirroring: https://github.com/w3c/wot-thing-description/blob/main/.prettierignore

### CODE REVIEW

**Review by lu-zero — APPROVED** [review]

File: .github/workflows/prettify.yml [hunk 1]
```diff
@@ -3,6 +3,16 @@ name: Prettifying
 on:
   pull_request:
     branches: [main]
+    paths-ignore:
+      - "publication/**"
+      - "errata.html"
+      - "errata11.html"
+      - "ontology/*.html"
+      - "index.html"
```
Comment (danielpeintner) [inline .github/workflows/prettify.yml:11]: Why do we exclude index.html ? It is auto-generated anyway...

File: .github/workflows/prettify.yml [hunk 2]
```diff
@@ -3,6 +3,16 @@ name: Prettifying
...
+      - "validation/tm-json-schema-validation.json"
```
Comment (danielpeintner) [inline .github/workflows/prettify.yml:12]: Not sure about this either and if we exclude this file we should exclude https://github.com/w3c/wot-thing-description/blob/main/validation/td-json-schema-validation.json also

File: .github/workflows/prettify.yml [hunk 3]
```diff
@@ -3,6 +3,16 @@ name: Prettifying
...
+      - "context/td-context-1.1.jsonld"
```
Comment (danielpeintner) [inline .github/workflows/prettify.yml:13]: Not sure again why we exclude just this file in the folder?

**Review by danielpeintner — COMMENTED** [review]
Some questions below...

File: .github/workflows/prettify.yml (hunk 2 above)
Comment (egekorkan) [reply .github/workflows/prettify.yml:12]: So the initial decision was to format only what we write and not format auto generated stuff. That is why this and index.html are ignored

Comment (egekorkan) [reply .github/workflows/prettify.yml:12]: See also https://github.com/w3c/wot-thing-description/issues/1926#issue-2006498787

These two comments should address all the questions

Comment (danielpeintner) [reply .github/workflows/prettify.yml:12]: I see the rational. Anyhow, the consequence is that once we update the tool that generates those other files might create huge diffs even though it didn't really change that much. Anyhow, I don't feel very strongly about it for now.

Comment (egekorkan) [reply .github/workflows/prettify.yml:12]: > we would directly not commit any generated file to the tree

We have to submit at least the index.html but the rest can live somewhere else.

> Anyhow, the consequence is that once we update the tool that generates those other files might create huge diffs even though it didn't really change that much

That is a good point. I am fine to format those as well, it wouldn't hurt to do so.

**Pull request merged**